test/zebras/test_schema.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+import numpy as np
+
+import gluonts.zebras as zb
+
+
+schema = zb.Schema(
+    {
+        "target": zb.Field(ndim=1, tdim=0, past_only=True),
+        "time_feat": zb.Field(ndim=2, tdim=-1, preprocess=np.atleast_2d),
+        "static_feat": zb.Field(ndim=1, preprocess=np.atleast_1d),
+    }
+)
+
+
+def test_schema_timeframe():
+    xs = [1, 2, 3, 4, 5]
+    row = {"target": xs, "time_feat": xs, "static_feat": 1}
+    schema.load_timeframe(row)
+
+    row = {"target": xs, "time_feat": [xs, xs], "static_feat": [1]}
+    schema.load_timeframe(row)
```

Comment:
Any meaningful assertion that could be added here, other than just loading the data?